### ISSUE TITLE
sentence.json() error

### ISSUE THREAD

**Issue by pavlo-vasylkivskyi-scx** [spencermountain/wtf_wikipedia#337]
Hello. I'm here again to tell you about a new issue that I recently found. I had made parsing of a few sentences from the topic with the title '[J. J. Abrams](https://en.wikipedia.org/w/index.php?title=J._J._Abrams&action=edit&oldid=933394946)' 
![image](https://user-images.githubusercontent.com/50583998/75255999-79c36100-57eb-11ea-925e-380a31385b2f.png)
![Screen Shot 2020-02-25 at 16 27 42](https://user-images.githubusercontent.com/50583998/75256300-ecccd780-57eb-11ea-916b-d795639d4325.png)

And I got these results 
![Screen Shot 2020-02-25 at 16 31 05](https://user-images.githubusercontent.com/50583998/75256628-65339880-57ec-11ea-92fe-4853ed12b051.png)

As you can see `sentence.json()` in this case works not so good. It would be wonderful if you can fix this. Thanks

**Comment by spencermountain** [comment]
oh hey, good find Pavlo - 
looks like the sentence parser gets tripped on the initial with a period, due to a sloppy regex [here](https://github.com/spencermountain/wtf_wikipedia/blob/master/src/04-sentence/parse.js#L10)

```js
let doc = wtf(`known as '''J. Abrams'''`)
console.log(doc.sentences(0).text())
```
i'll fix this on dev today, but next release will be behind a few other features, so may be two or so weeks.
cheers

**Comment by spencermountain** [comment]
if you wanted to fork a quick fix, it would be
```js
const acronym_reg = new RegExp("[ |.|'|[][A-Z].? *?$", 'i')
```

**Comment by pavlo-vasylkivskyi-scx** [comment]
Thank you very much!

**Comment by spencermountain** [comment]
fixed in v8.0.0
cheers
